namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "application" {
+  default = "probation-tiering"
+}
+
+variable "namespace" {
+  default = "probation_tiering_dev"
```

Comment:
this needs to be `probation-tiering-dev` - it's the namespace name, and that's hyphenated.